charts/steward/values.yaml

```diff
@@ -80,10 +80,8 @@ pipelineRuns:
       # Example: http://elasticsearch-master.elasticsearch.svc.cluster.local:9200/jenkins-logs/_doc
       indexURL: ""
   jenkinsfileRunner:
-    image:
-      repository: stewardci/stewardci-jenkinsfile-runner
-      tag: "200921_6cc247f"
-      pullPolicy: IfNotPresent
+    image: "stewardci/stewardci-jenkinsfile-runner:200921_6cc247f"
```

Comment:
We need to make sure this is merged correctly. I assume the image version changed in the master, and hope it will cause a conflict.

Reply:
Currently it is the same image as in the master branch.